source/NVDAObjects/window/scintilla.py

```diff
@@ -193,9 +194,9 @@ def _getLineOffsets(self,offset):
 			# Lines in Scintilla refer to document lines, not wrapped lines.
 			# There's no way to retrieve wrapped lines, so use screen coordinates.
 			y=watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POINTYFROMPOSITION,None,offset)
-			top,left,width,height=self.obj.location
-			start = self._getOffsetFromPoint(0,y)
-			end=self._getOffsetFromPoint(width,y)
+			location=self.obj.location
+			start = self._getOffsetFromPoint(location.left, location.top+y)

```

Comment:
If you use _getPointFromOffset above, this will be:

```suggestion
			start = self._getOffsetFromPoint(location.left, y)
```